accounts/README.md

```diff
@@ -1,10 +1,121 @@
+
+## API Specification
+
+### General requirements
+
+The Account MUST append the 20 bytes of the `msg.sender` address to the calldata to each Account configuration-related method (setting a hook,
+adding a plugin, etc., for the full list, refer to [Manager's](../manager/README.md) specification). This is required because
+to support some functionality, such as function handlers, an Account may be required to execute a CALL
+operation that potentially bypasses the authorization to the Manager contract. This is required to allow the Manager to identify the transaction's sender. It is assumed
+that the only way to have the `msg.sender` equal to the Account address in the call frame is to execute a call through the Account's built-in authorization scheme.
+
+### Expected interface per Safe{Core} Protocol Module
+
+#### Plugins
+
+The account MUST implement the following interface:
+
+```Solidity
+interface IAccount {
+    function execTransactionFromModuleReturnData(
+        address to,
+        uint256 value,
+        bytes memory data,
+        uint8 operation
+    ) external returns (bool success, bytes memory returnData);
+}
+```
+
+where:
+
+| Type         | Name      | Description                                                                                                                                                                                                                                                                                                                                                                               |
+|--------------|-----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| address      | to        | The address of the contract to be called                                                                                                                                                                                                                                                                                                                                                  |
+| uint256      | value     | The amount of Native Token to be sent                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
I wouldn't capitalise "native token" here.

Also, something that is not obvious (and related to EVM semantics) is that this value can't be specified when doing a `DELEGATECALL` (since you don't send values with `DELEGATECALL` and it inherits the current transaction value).

Actually, thinking about it, how _do_ you do a `DELEGATECALL` with a value under 4337? I assume it that you would have to make the `Safe` call itself with a value to do the actual `DELEGATECALL` you want?

Reply:
> Also, something that is not obvious (and related to EVM semantics) is that this value can't be specified when doing a DELEGATECALL (since you don't send values with DELEGATECALL and it inherits the current transaction value).

I specified that the value would be ignored if the operation is delegatecall.

> Actually, thinking about it, how do you do a DELEGATECALL with a value under 4337? I assume it that you would have to make the Safe call itself with a value to do the actual DELEGATECALL you want?

🤯 Something like this, yeah 😂